src/components/Chat/ColorPicker.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from 'react';
+import {CirclePicker} from 'react-color';
+interface ColorPickerProps {
+  color: string;
+  onUpdateColor: (color: string) => void;
+}
+const ColorPicker: React.FC<ColorPickerProps> = (props) => {
+  const [isActive, setActive] = useState(false);
```

Comment:
Optional suggestion: This project has `react-use` as a dependency; you can use `const [isActive, toggleIsActive] = useToggle()` for convenience